src/EnergyPlus/StandardRatings.cc

```diff
@@ -1318,15 +1327,21 @@ namespace StandardRatings {
                                NetCoolingCapRated(ns),
                                SEER_User * ConvFromSIToIP,
                                SEER_Standard * ConvFromSIToIP,
-                               0.0,
-                               0.0,
+                               EER,
+                               EER * ConvFromSIToIP,
                                0.0,
                                0.0,
                                0.0,
                                0.0,
                                0,
                                false);
 
+            Real64 EER_Report(0.0);
+            if (SEER2_Standard > 0.0) {
+                EER_Report = EER;
+            } else if (IEER_2022 > 0.0) {
+                EER_Report = EER_2022;
+            }
             // Writes the SEER value to the EIO file and standard tabular output tables
             ReportDXCoilRating(state,
```

Comment:
As we wrote above (for your comment about the table 'DX Cooling Coil Standard Rating Information'), the 'DX Cooling Coil Standard Rating Information' table is old and not structured to include ratings from different versions of standard.  This is why we suggested to Edwin, that this table be removed, as it is (effectively) replaced by the 'DX Cooling Coil Standard Ratings 2017' and '... 2024' tables.  These two new tables are designed to be specific and unambiguous.  We had proposed header changes to address SEER vs SEER2, but pulled those in our effort to be included in the 24.1 release.  We could restore them if we are destined for 24.2 instead.
Regarding the value differences in the 2023 table for UnitarySystem_VSCoolingCoil.idf:
We agree that the new SEER2 values look suspect for this Coil, so we will re-checked the code (seems correct) and also generates the SEER2 values in the NIST Excel Workbook as a 3rd party validation check.  The new value is within 3% of NIST, so it looks like this coil has low performance.  Other sample files with 2-speed CurveFit coils have SEER2 in the range of 12 to 13.
We DO expect the values to change between 23.2 and 24.1, as we made significant changes to the calculation for Variable & CurveFit DX coils, in order to be true to the standard.  We learned a lot between the 22.1 cycle (original) and the 24.1 cycle.

Reply:
The pattern of two sets of output with a single header row was introduced in #9480 in v22.2. The need for a different header should have been caught back then. The simplest solution is to add a second flag value and print the correct header the first time `ReportDXCoilRating` is called with `AHRI2023StandardRatings=true`. The second header and row identifier could be "DX Cooling Coil AHRI 2023 Standard Rating Information". This would be consistent with `ReportDXCoolCoilDataCenterApplication`.

Regarding the change in results for UnitarySystem_VSCoolingCoil, with this branch, the 2017 and 2023 values for the same coil are very different now. That's quite surprising, but if that's expected, then ok.
![image](https://github.com/NREL/EnergyPlus/assets/4991075/e8b5bbfa-447c-43fa-b8bf-c0d8124e06c0)

I don't see the same magnitude of difference with MultiSpeedHeatPump.idf
![image](https://github.com/NREL/EnergyPlus/assets/4991075/85e6c197-a230-4f95-83fd-bd0bb73570e9)

Reply:
You are correct that we introduced 2017 & 2023 rows in the "DX Cooling Coil Standard Rating Information" table.  At that time, we could not figure out how to make two headers in the same table. Our apologies for not getting it right in 22.1.  However, we are reporting the same information in the first of two tables in your screenshots above (ending in "... Standard Ratings 2017").  This table existed when we started, so I think it was originally added by Bereket. We simply added the "2017" to so make clear that the calculations shown in that table are from the 2017 version of the standard.  We then added the table for the 2023 version calculations, so that comparison between versions is clear.  Since our feature has been pushed to the 24.2 cycle, we will add back the header changes (which differentiate COP/COP2, EER/EER2, and SEER/SEER2) as shown below:
![image](https://github.com/NREL/EnergyPlus/assets/8195933/e5e0ab2f-aff1-42ff-bd8e-09f6e7f7f33b)
Given that the values reported in the "... Standard Rating Information" table are included in the "... Standard Ratings 2017" table, isn't it reasonable that the older one be deprecated?
Regarding the SEER/SEER2 value difference for "UnitarySystem_VSCoolingCoil.idf" file -- we looked deeper into this. Please excuse us if we step on anyone's toes with the following: we believe that the difference stems from this being a CurveFit coil.  The 2017 implementation for all CurveFit coils appears to be based on the 2017 implementation for SingleSpeed coils -- using a single calculation path.  The 2017 standard (pages 52/53) defines four different calculation alternatives, based on the relative values for cooling capacity and building load at each of 8 bin temperatures.  This is what we implemented in the 2023 calculations.
For reference, the SEER-2017 implementation for all CurveFit coils is here: https://github.com/RichSee/EnergyPlus/blob/working-24-1/src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc#L516C8-L540C10  
And the SEER2-2023 implementation for TwoSpeed CurveFit coils is here: https://github.com/RichSee/EnergyPlus/blob/working-24-1/src/EnergyPlus/StandardRatings.cc#L4635C15-L4676C14  
We are in process of logging values from these calculations in order to demonstrate why they are different. At this point, we believe this is because the 2017 implementation for CurveFit coils is derived from the calculation for SingleSpeed coils.
The reason the values are very similar for "MultiSpeedHeatPump.idf" is because the 2017 implementation for MultiSpeed coils does follow the standard, as does the 2023 implementation.  Hence the values are quite similar.
Is it possible to schedule a call to discuss these things, and make a plan to resolve them?

Reply:
> Given that the values reported in the "... Standard Rating Information" table are included in the "... Standard Ratings 2017" table, isn't it reasonable that the older one be deprecated?

No, they serve two different purposes. The eio output is comma separated and easier for interfaces to grab the information from compared to the html table output which is more human-readable. (The eio output can also show up in the table output as the Initialization Summary report, but that's generated automatically from the eio contents.) I'm not sure this is worth much discussion, it's roughly five-line fix to change the header written to the eio file for the newer ratings and to track two onetime flags. If that's not clear, I can push those changes myself.

>  the 2017 implementation for CurveFit coils is derived from the calculation for SingleSpeed coils.

I'm not sure who implemented what here, but the ratings for the CurveFit coil need to be sensitive to the coil's number of speeds which can be 1 or more. If the results above are indeed single-speed for 2017 and multi-speed for 2023, then I'm still puzzled, because I would expect a multi-speed SEER to be higher than a single-speed SEER, not significantly lower.  For reference here, the coil in UnitarySystem_VSCoolingCoil is Coil:Cooling:DX:CurveFit:Performance with 2 speeds.

Reply:
Why is 2023 COP one-half of the other 2 standard methods when COP is at "full speed" capacity and rated capacity has only a small change? That would mean full load power increased by a factor of 2 simply by changing the rating method.

Reply:
@rraustad  -- Thank you for raising this issue.  We report COP2 for Full Speed 'Test Condition' (q_A_Full/p_A_Full). We double-checked our calculation of COP2, and found that, for 2-Speed CurveFit coils, Speed-1 was misinterpreted as the HighSpeed and Speed-2 was misinterpreted as LowSpeed (which is the order for "TwoSpeed" DX coils).  This has been corrected and the resulting COP2 values are now consistently tracking with those from the previous implementation(s), across several test files.

Reply:
@RichSee eio output looks good. Thanks.